cloud_platform_azure/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2017-2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+
+{
+    "name": "Cloud Platform Azure",
+    "summary": "Addons required for the Camptocamp Cloud Platform on Azure",
+    "version": "13.0.1.0.0",
+    "author": "Camptocamp,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Extra Tools",
+    "depends": [
+        "cloud_platform",
+        "attachment_azure",
+        "monitoring_prometheus",
+    ],
+    "excludes": [
```

Comment:
just wondering does it has any impact or just for information

Reply:
It will show an error in the logs and prevent the installation if you try to install the module when any of those is already installed